linode/manager
# M3-5177: Use React Query for Account Settings

## Description

- Remove `/v4/account/settings` data from Redux
- Uses **React Query**'s  _Query_ and _Mutation_ for `/v4/account/settings` 

## How to test

- Test all general use of Cloud Manager. Things that are most likely to break include `backups`, `object storage`, `managed`, `longview` and `linode network helper`.
- Ensure all everything at [`/account/settings`](http://localhost:3000/account/settings) still work as expected
